### ISSUE TITLE
adding product to cart without going to details page

### ISSUE THREAD

**Issue by fimdomeio** [prgagarina167/bitnost-templates#5]
The block of each release in the table. Currently we see a dummy with information. Basically there has to be a possibility to add the product to cart from the listing without moving to the details page!

a) Here might be 2 ways – to show buy button to for both formats like at http://www.piccadillyrecords.com/counter/catalogue.php?genre=35, or anost.net

b) We just show one buy button to the format filtered above and always split the lists, CDs and LPs separately. Hence we'll need one buy button per block and a second one for navigating to details page.

**Comment by fimdomeio** [comment]
Tried one way to do it. It shows a bootstrap popover if theres more than one choice or adds the record type to the button if there's only one choice. Let me know what you think.

**Comment by prgagarina167** [comment]
One comment for popover – it has to be 100% clear which product format and price you're adding to cart. Currently (or let's say in general), when there are more than one format it's not clear from layout – I think this has to be changed – "Add to cart" button has to be paired with particular format/price visually.

As for general styling, overall really liking it!

**Comment by prgagarina167** [comment]
The list of links that have varied width and cart icon in the end is not working really well. Functionally this part is OK. However:

- links are too small/dense, so it's difficult to distinguish between format, price, euro sign etc. This probably doesn't work really well being written into one line.

- we need one clear Button, that says BUY or ADD to Basket
- the cart icon – let's dump from release block completely. it might stay only in the header, but I would still prefer word Basket or Cart instead
- price might go as text, perhaps of the bigger size – but should not have any other unnecassary noisy elements around it. 15.99 EUR or 13.99 € .
- format: perhaps we should make it distinct from other text, diff background, or dash or slash. See boomkat/anost/piccadillyrec for examples.

Basically the point is to have a good visual distinction between these infos. Big and clear, green button or whatever! The size of this information might be increased too.


![screen shot 2015-04-20 at 11 58 13 am](https://cloud.githubusercontent.com/assets/1527960/7226814/92a6bd64-e754-11e4-9d0b-3434df5f49a1.png)

**Comment by fimdomeio** [comment]
![captura de ecra 2015-04-21 as 10 46 26](https://cloud.githubusercontent.com/assets/705133/7249643/bd02c3fe-e813-11e4-8bb7-df1f5a3b4236.png)
I reorganized the info a little bit. I think it adresses all these issues.

**Comment by prgagarina167** [comment]
That's better thanks! Can we have a proper button instead of link and change text into BUY?
